distribution/test/.editorconfig

```diff
@@ -20,6 +20,7 @@
 # Meziantou
 # https://www.meziantou.net/enforcing-asynchronous-code-good-practices-using-a-roslyn-analyzer.htm
 dotnet_diagnostic.MA0004.severity = none            # https://github.com/atc-net/atc-coding-rules/blob/main/documentation/CodeAnalyzersRules/Meziantou/MA0004.md
+dotnet_diagnostic.MA0016.severity = none            # MA0016: Prefer return collection abstraction instead of implementation
```

Comment:
We need to have a href to the decision .md file and the severity should match the decision.

Reply:
Feel free to use the suggestion feature here another time to make the review process easier for all involved :)